gen3-client/g3cmd/root.go

```diff
@@ -5,16 +5,14 @@ import (
 	"log"
 	"os"
 
-	homedir "github.com/mitchellh/go-homedir"
 	"github.com/spf13/cobra"
-	"github.com/spf13/viper"
 	latest "github.com/tcnksm/go-latest"
+	"github.com/uc-cdis/gen3-client/gen3-client/jwt"
 	"github.com/uc-cdis/gen3-client/gen3-client/logs"
 )
 
-var cfgFile string
 var profile string
-var uri string
+var profileConfig jwt.Credential
```

Comment:
just curious, is this kind of global variable across files a normal practice / good practice in Go?
i find it difficult to read, with the declaration in `root`, initializations in `init()` and uses all over the place 👀

Reply:
no it is actually not 😅 global vars are dangerous in Go as well as in other languages

for the client it is kind of special since it is a standalone application and can execute one command at a time, so many downside of the global vars are not so effecitive... and it make the refactor work a bit easier...

Reply:
✅ thanks!